### PR TITLE
[devtools-translate] Korean - What's New in DevTools (Chrome 98)

### LINKED ISSUE #1959: [devtools-translate] Korean - What's New in DevTools (Chrome 98)
Post: https://developer.chrome.com/blog/new-in-devtools-98/

Translator: @TORU0239
Reviewers: @cwdoh @techhtml

Translated by: Jan 25

### PR DESCRIPTION
**Pull request by TORU0239**
Fixes #1959

Changes proposed in this pull request:

- added translation in Korean

### CODE REVIEW

File: site/ko/blog/new-in-devtools-98/index.md [hunk 1]
```diff
@@ -61,94 +67,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
-
+## 뒤로/앞으로 캐쉬 탭을 이용해 페이지가 캐쉬 가능한지 확인하기 {: #bfcache }
 <!-- [Back/forward cache (or bfcache)](https://web.dev/bfcache/) is a browser optimization that enables instant back and forward navigation.  -->
+[뒤로/앞으로 캐쉬(혹은 bfcache)](https://web.dev/bfcache/)는 브라우저 최적화의 하나로, 즉각적인 앞뒤 내비게이션을 가능하게 해 주는 기능입니다.
 
 <!-- The new **Back/forward cache** tab can help you test your pages to ensure they're optimized for bfcache, and identify any issues that may be preventing them from being eligible. -->
+새롭게 선보이는 **뒤로/앞으로 캐쉬** 탭을 사용하여, 웹 페이지들이 bfcache에 대해 최적화되었는지를 테스트할 수 있으며, 또한 페이지가 bfcache를 사용하지 못하도록 하는 이슈들을 파악할 수 있습니다.
 
 <!-- To test a particular page, navigate to it in Chrome and then in DevTools go to **Application** > **Back-forward Cache**. Next, click the **Test back/forward cache** button and DevTools will attempt to navigate away and back to determine whether the page could be restored from bfcache. -->
+특정 페이지를 테스트하기 위해, 크롬에서 해당 페이지로 이동한 다음 DevTools에서 **애플리케이션** > **뒤로-앞으로 캐쉬** 를 선택합니다. 그리고 ** 뒤로/앞으로 캐쉬 테스트** 버튼을 클릭하여 DevTools가 앞으로 혹은 뒤로 페이지 탐색을 시작하도록 하여, 해당 페이지가 bfcache 로부터 복구 가능한지를 확인할 수 있습니다.
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-98/index.md:98]:
```suggestion
특정 페이지를 테스트하기 위해, 크롬에서 해당 페이지로 이동한 다음 DevTools에서 **애플리케이션** > **뒤로-앞으로 캐쉬** 를 선택합니다. 그리고 **뒤로/앞으로 캐쉬 테스트** 버튼을 클릭하여 DevTools가 앞으로 혹은 뒤로 페이지 탐색을 시작하도록 하여, 해당 페이지가 bfcache 로부터 복구 가능한지를 확인할 수 있습니다.
```

File: site/ko/blog/new-in-devtools-98/index.md [hunk 2]
```diff
@@ -157,73 +184,96 @@ Chromium issue: [1050021](https://crbug.com/1050021)
...
  
 <!-- ### Keyboard shortcut to expand all search results {: #toggle-search-result } -->
+### 모든 검색 결과 키보드 단축키 {: #toggle-search-result }
 
 <!-- A new shortcut is added in the **Search** tab allowing you to expand and collapse all the search results. Previously, you could only expand and collapse the search results by clicking on one file at a time. -->
+**검색** 탭에 모든 검색 결과를 열고 닫을 수 있는 단축키가 추가되었습니다. 이전에는 한 번에 하나의 파일을 클릭해야만 검색 결과를 확장 및 축소할 수 있었습니다.
 
 <!-- Open the search tab via **Esc** > **3-dot** menu > **Search**. Enter a search string (e.g. function) and press **Enter** to see the list of search results. Focus on the search results and use the following shortcut to expand/collapse the search files: -->
 
+*Esc** > **점 3개 메뉴** > **검색** 을 통해 검색 탭을 엽니다. 검색하고자 하는 스트링(예: function)을 입력하고 **Enter**를 눌러서 검색 결과 리스트를 볼 수 있습니다. 검색 결과에 포커싱을 주고 다음 단축키를 사용하여 검색된 파일들을 확장 및 축소할 수 있습니다.
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-98/index.md:194]:
```suggestion
**Esc** > **점 3개 메뉴** > **검색** 을 통해 검색 탭을 엽니다. 검색하고자 하는 스트링(예: function)을 입력하고 **Enter**를 눌러서 검색 결과 리스트를 볼 수 있습니다. 검색 결과에 포커싱을 주고 다음 단축키를 사용하여 검색된 파일들을 확장 및 축소할 수 있습니다.
```

**Review by chybie — COMMENTED** [review]
Fix linting issues

File: site/ko/blog/new-in-devtools-98/index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ tags:
 draft: true
 ---
 
-*이 게시글의 번역에는 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 참여하였으며, [최원영](https://www.linkedin.com/in/toruchoi)님과 [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*
+*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하였으며, [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님과 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 리뷰를 맡아 주셨습니다.*
 
 {% include 'partials/devtools/ko/banner.md' %}
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-98/index.md:21]:
```suggestion
{% include 'partials/devtools/ko/banner.md' %}
```

{% YouTube id='YqkIS88VulM' %}

File: site/ko/blog/new-in-devtools-98/index.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ tags:
 draft: true
 ---
 
-*이 게시글의 번역에는 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 참여하였으며, [최원영](https://www.linkedin.com/in/toruchoi)님과 [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*
+*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하였으며, [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님과 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 리뷰를 맡아 주셨습니다.*
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-98/index.md:20]: {% YouTube id='YqkIS88VulM' %}

File: site/ko/blog/new-in-devtools-98/index.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ tags:
...
 
 {% include 'partials/devtools/ko/banner.md' %}
```
Comment (chybie) [inline site/ko/blog/new-in-devtools-98/index.md:22]:
```suggestion

{% YouTube id='YqkIS88VulM' %}
```

**Review by cwdoh — COMMENTED** [review]
I thinks this translation is almost good to go.

**Comment by cwdoh** [comment]
After the Korean mode support, it seems necessary for us to discuss the issue of the English / Korean strings such as commands. [If there's translations for selected locale from DevTools, command menu works by only translated string](https://github.com/GoogleChrome/developer.chrome.com/pull/2006#discussion_r795206589). Do we need to show both of them?

File: site/ko/blog/new-in-devtools-98/index.md [hunk 6]
```diff
@@ -29,29 +30,35 @@ draft: true
...
-
+## 미리보기 기능: 전체 페이지 접근성 트리 {: #a11y-tree }
 <!-- The new **Full-page accessibility tree** makes it easier for you to get an overview of the full-page [accessibility tree](/blog/full-accessibility-tree/#what-is-the-accessibility-tree) and help you better understand how your web content is exposed to assistive technology.  -->
+사용자들은 새로운 **전체 페이지 접근성 트리**를 통해 전체 페이지 [접근성 트리](/blog/full-accessibility-tree/#what-is-the-accessibility-tree)의 개요를 좀더 쉽게 알 수 있으며, 이 기능은 웹 컨텐츠가 어떻게 보조 기술에 노출되는지를 사용자들이 좀더 잘 이해할 수 있도록 합니다.
 
 <!-- In the **Elements** panel, open the **Accessibility** pane and select **Enable full-page accessibility tree** checkbox. Then, reload DevTools and you will see a new accessibility button in the **Elements** panel. -->
+**요소** 패널에서 **접근성** 패널을 열고 **전체 페이지 접근성 트리 사용 설정** 체크박스를 선택합니다. 이후 DevTools를 새로고침하면 **요소** 패널에 접근성 버튼이 생긴 것을 확인할 수 있습니다.
 
 <!-- Click on it to toggle to the **Full-page accessibility tree** view. You can expand nodes or click to see details in the  **Accessibility** pane. -->
+**전체 페이지 접근성 트리**뷰로 전환하기 위해서, 아래 그림의 왼쪽 아이콘 위에서 클릭합니다. 우리는 각 노드들을 확장할 수 있으며, 각자의 노드를 클릭하여 **접근성** 패널에서 세부 사항들을 볼 수 있습니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:41]: It worked well for me when using sublime text as well as preview model in github though, and think adding whitespace should be better

File: site/ko/blog/new-in-devtools-98/index.md [hunk 7]
```diff
@@ -29,29 +30,35 @@ draft: true
...
 
 Chromium issue: [887173](https://crbug.com/887173)
 
 
 <!-- ## More precise changes in the Changes tab {: #changes }  -->
-
+## 변경 탭에서의 더욱 정교한 변화들 {: #changes }
 <!-- The code changes in the **Changes** tab is pretty-printed automatically.  -->
-
+**변경** 탭에서의 코드 체인지들은 자동적으로 화사하게 프린트됩니다. <!-- 문맥에 맞는 단어 선택 필요 -->
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:59]: Applied

File: site/ko/blog/new-in-devtools-98/index.md [hunk 8]
```diff
@@ -29,29 +30,35 @@ draft: true
...
 <!-- Previously, it was hard to trace the actual changes of minified source code because all the code is shown in a single line.  -->
+이전 버전에서는 모든 코드가 한 줄에 표시되었기 때문에 축소된 소스코드에서의 실제 변화를 추적하는 것은 쉽지 않았습니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:61]: Will go for `압축된`

File: site/ko/blog/new-in-devtools-98/index.md [hunk 9]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
+이제 여러분들은 [Recorder](/docs/devtools/recorder/)에서 모든 단계 혹은 특정한 단계에서의 **타임아웃** 설정을 조정할 수 있게 됐습니다. 이 기능은 느린 네트워크 요청과 긴 애니메이션이 포함된 페이지에서 유용하게 사용될 것입니다.
 
 <!-- For example, I [recorded a user flow](/docs/devtools/recorder/#record) on this [demo page](https://jec.fyi/demo/pup-slow-result) to load and click on the menu item. However, the loading of the menu items is slow (it takes 6 seconds). The [replay](/docs/devtools/recorder/#replay) of this user flow failed because it exceeds 5 seconds  (the default timeout). -->
+예를 들어, [데모 페이지](https://jec.fyi/demo/pup-slow-result)에서 [유저 플로우 녹화](/docs/devtools/recorder/#record) 를 시작하고 메뉴 항목을 클릭하여 로드합니다. 그러나 메뉴를 불러들이는 것이 6초가 걸렸습니다. 로딩 시간이 5초를 초과-기본 타임아웃-했기 때문에 이 유저 플로우의 [다시보기](/docs/devtools/recorder/#replay)는 정상적으로 동작하지 않았습니다.
 
 <!-- We can use the new **Timeout** settings to fix this. Expand the step which we click on the menu item. [Edit the step](/docs/devtools/recorder/#edit-steps) by  **Add timeout** and set it to **6000** milliseconds (equal to 6s). -->
 
+이러한 문제를 해결하기 위해, 여러분은 새롭게 추가된 **Timeout** 설정을 사용할 수 있습니다. 메뉴 항목에서 우리가 클릭한 스텝을 확장합니다. **타임아웃 추가** 를 이용해 [스텝을 편집](/docs/devtools/recorder/#edit-steps)하고 타임아웃을 **6000** 밀리초 (6초)로 설정합니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:80]: Applied `타임아웃`

File: site/ko/blog/new-in-devtools-98/index.md [hunk 10]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
+
 <!-- Optionally, you can adjust the **Timeout** in the **Replay settings** for all the steps. Expand the **Replay settings** and edit the **Timeout** value.  -->
+추가적으로, 여러분은 모든 단계에 대한 **Timeout**을 **Replay settings**에서도 조정할 수 있습니다. **Replay settings**을 확장한 다음 **Timeout** 값을 수정합니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:83]: Modified to `타임아웃`

File: site/ko/blog/new-in-devtools-98/index.md [hunk 11]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/y7RDpIp3pd2n6Vnxc5Du.png", alt="유저 플로우 녹화를 위한 타임아웃 설정", width="800", height="530" %}
 
 Chromium issue: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Ensure your pages are cacheable with the Back/forward cache tab {: #bfcache } -->
-
+## 뒤로/앞으로 캐쉬 탭을 이용해 페이지가 캐쉬 가능한지 확인하기 {: #bfcache }
 <!-- [Back/forward cache (or bfcache)](https://web.dev/bfcache/) is a browser optimization that enables instant back and forward navigation.  -->
+[뒤로/앞으로 캐쉬(혹은 bfcache)](https://web.dev/bfcache/)는 브라우저 최적화의 하나로, 즉각적인 앞뒤 내비게이션을 가능하게 해 주는 기능입니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:93]: :)

File: site/ko/blog/new-in-devtools-98/index.md [hunk 12]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
 
 
 <!-- ## Emulate the CSS forced-colors media feature {: #forced-colors } -->
+ ## CSS의 forced-colors 미디어 기능 에뮬레이팅 {: #forced-colors }
 
 <!-- The [forced-colors](https://drafts.csswg.org/mediaqueries-5/#forced-colors) CSS media feature is used to detect if the user agent has enabled a forced colors mode (e.g. Windows High Contrast mode) where it enforces a user-chosen limited color palette on the page.  -->
+User agent 가 웹페이지에서 사용자가 선택한 제한된 색 팔레트를 사용하도록 강제하는 forced-colors 모드 (예: 윈도우즈 하이 콘트라스트 모드)를 활성화했는지의 여부를 CSS의 [forced-colors](https://drafts.csswg.org/mediaqueries-5/#forced-colors) 미디어 기능을 통해 알아볼 수 있습니다.
 
 <!-- Open the [Command Menu](/docs/devtools/command-menu/), run the **Show Rendering** command, and then set the **Emulate CSS media feature forced-colors** dropdown. -->
+[명령 메뉴](/docs/devtools/command-menu/) 를 열고 **Show Rendering** 명령어를 실행한 다음, **Emulate CSS media feature forced-colors** 드롭다운 메뉴를 설정할 수 있습니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:137]: Applied `렌더링 표시`

File: site/ko/blog/new-in-devtools-98/index.md [hunk 13]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/75qGjkzfbXfOEJUhML5i.png", alt="CSS forced-colors media feature", width="800", height="623" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/75qGjkzfbXfOEJUhML5i.png", alt="CSS forced-colors 미디어 기능", width="800", height="623" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/db79deee160cda92eda91775a27773611dce8188 #}
 
 Chromium issue: [1130859](https://crbug.com/1130859)
 
 <!-- ## Show rulers on hover command {: #show-rulers } -->
+## 마우스를 가져가면 눈금자 표시 {: #show-rulers }
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:146]: How about `커서를 올렸을 때 눈금자 표시하기`? followed it

File: site/ko/blog/new-in-devtools-98/index.md [hunk 14]
```diff
@@ -61,94 +68,115 @@ Chromium issues: [1238818](https://crbug.com/1238818), [1268754](https://crbug.c
...
 
 <!-- You can now open the [Command Menu](/docs/devtools/command-menu/) and run the **Show rulers on hover** command. The page rulers make it easier to measure the width and height of an element. -->
+여러분은 이제 [Command Menu](/docs/devtools/command-menu/) 를 열고 **마우스를 가져가면 눈금자 표시**를 실행할 수 있습니다. 페이지의 눈금 기능을 이용하여 우리는 각 요소의 길이와 넓이를 좀더 쉽게 측정할 수 있습니다.
```
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:149]: Reverted to `Show rulers on hover` for the paragraph, as well as `alt`

File: site/ko/blog/new-in-devtools-98/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md]: Screenshot 2022-01-31 at 11 55 36 PM

I just did screenshot with regards to this translation. should keep the original translation

File: site/ko/blog/new-in-devtools-98/index.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (TORU0239) [reply site/ko/blog/new-in-devtools-98/index.md:149]: For the same reason, I think better use the original translation

**Review by chybie — APPROVED** [review]

**Comment by chybie** [comment]
Thank you all!

**Pull request merged**